packages/notification-service/test/transfers.test.ts

```diff
@@ -116,30 +135,34 @@ describe('Transfers', () => {
 
   it('should exclude exchanges', () => {
     const celoTransfers = new Map<string, Transfer[]>()
-    const stableTransfers = new Map<string, Transfer[]>()
+    const cUsdTransfers = new Map<string, Transfer[]>()
+    const cEurTransfers = new Map<string, Transfer[]>()
 
     celoTransfers.set(CELO_EXCHANGE.txHash, [CELO_EXCHANGE])
-    stableTransfers.set(DOLLAR_EXCHANGE.txHash, [DOLLAR_EXCHANGE])
+    cUsdTransfers.set(DOLLAR_EXCHANGE.txHash, [DOLLAR_EXCHANGE])
+    cEurTransfers.set(DOLLAR_EXCHANGE.txHash, [EURO_EXCHANGE])
 
-    const concated = filterAndJoinTransfers(celoTransfers, stableTransfers)
+    const concated = filterAndJoinTransfers(celoTransfers, cUsdTransfers, cEurTransfers)
 
     expect(concated).toEqual([])
   })
 
   it('should include unique transactions and update the last block', () => {
     const celoTransfers = new Map<string, Transfer[]>()
-    const stableTransfers = new Map<string, Transfer[]>()
+    const cUsdTransfers = new Map<string, Transfer[]>()
+    const cEurTransfers = new Map<string, Transfer[]>()
 
     celoTransfers.set(CELO_TRANSFER.txHash, [CELO_TRANSFER])
-    stableTransfers.set(DOLLAR_TRANSFER.txHash, [DOLLAR_TRANSFER])
+    cUsdTransfers.set(DOLLAR_TRANSFER.txHash, [DOLLAR_TRANSFER])
+    cEurTransfers.set(EURO_TRANSFER.txHash, [EURO_TRANSFER])
 
-    const concated = filterAndJoinTransfers(celoTransfers, stableTransfers)
+    const concated = filterAndJoinTransfers(celoTransfers, cUsdTransfers, cEurTransfers)
 
-    expect(concated).toEqual([CELO_TRANSFER, DOLLAR_TRANSFER])
+    expect(concated).toEqual([CELO_TRANSFER, DOLLAR_TRANSFER, EURO_TRANSFER])
   })
 
   it('should notify for new transfers since last block notified', async () => {
-    const transfers = [CELO_TRANSFER, DOLLAR_TRANSFER]
+    const transfers = [CELO_TRANSFER, DOLLAR_TRANSFER, EURO_TRANSFER]
     const returned = await notifyForNewTransfers(transfers)
 
     expect(sendPaymentNotificationMock).toHaveBeenCalledWith(
```

Comment:
We should check `sendPaymentNotificationMock` has been called for the cEUR transfer.